src/udev/net/link-config.c

```diff
@@ -432,6 +433,16 @@ int link_config_apply(link_config_ctx *ctx, link_config *config,
         if (r < 0)
```

Comment:
hmm i think the ethtool calls above probably should not be repeated on anything but add/bind either, otherwise people might be confused if they issue "ethtool" to change something and then something eventually issues "udevadm trigger" and it is all reset...

Reply:
(hmm, i guess all these ethtool commands probably should be split out into their own helper function btw, it's so many now)